tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/XUnitImpactedTests.cs

```diff
@@ -86,6 +86,8 @@ public Task DisabledByEnvVar(string packageVersion)
         [Trait("Category", "TestIntegrations")]
         public Task EnabledBySettings(string packageVersion)
         {
+            Skip.If(EnvironmentHelper.IsAlpine(), "This test is currently flaky in alpine due to a Detached Head status. An issue has been opened to handle the situation. Meanwhile we are skipping it.");
```

Comment:
Just to clarify it _only_ seems to happen on arm64 too. Which is... weird... but this covers it so for now: 😄 :shipit: